src/lib/ui/layout/DetailRow.tsx

```diff
@@ -2,15 +2,22 @@ import type { PropsWithChildren } from 'react'
 
 interface DetailRowProps {
   label: string
+  sublabel?: string
 }
 
 export function DetailRow({
   label,
+  sublabel,
   children,
 }: PropsWithChildren<DetailRowProps>): JSX.Element {
   return (
     <div className='seam-detail-row'>
-      <p className='seam-row-label'>{label}</p>
+      <div className='seam-detail-row-label-wrap'>
+        <p className='seam-row-label'>{label}</p>
+        {sublabel != undefined && (
```

Comment:
Using `null` here will make the element show up in the DOM, messing up the layout of rows that don't have this property. This is also a flaw in the update to `DetailSection` with the optional `label` prop—the element is still in the DOM.